content/index.js

```diff
@@ -241,6 +247,27 @@ cli
       options.foldersearch = newFoldersearch;
     }
 
+    // We might need to parse the list of files because it's a string
+    // with a space separator.
+    if (options.files) {
+      // The get-diff-action will make this a massive string that looks like
+      // this: `'content/files/en-us/a/index.html','content/files/en-us/a/index.html'`
+      // so we need to turn that into an array:
+      // ["content/files/en-us/a/index.html", "content/files/en-us/b/index.html"]`
+      // Note, when you use get-diff-action in GitHub Actions, it's a comma
+      // but if you use the manualy `git diff --name-only ...` on your command
+      // line it's a newline.
+      options.files = options.files.split(/[,\n]/).map((item) => {
+        if (
+          (item.startsWith("'") || item.startsWith('"')) &&
+          item.charAt(0) === item.charAt(item.length - 1)
+        ) {
+          item = item.slice(1, item.length - 1);
+        }
+        return item;
```

Comment:
Nit. I couldn't help myself, but the following would do the exact same thing too:
```suggestion
        // Remove any single or double-quote bookends.
        return item.replace(/^(['"])(.*)\1$/, "$2")
```

Reply:
Fancy!

Reply:
Fancy!